src/drc/api/views/enkelvoudig_informatieobject.py

```diff
@@ -305,35 +284,19 @@ def download(self, request, *args, **kwargs):
             mimetype="application/octet-stream",
         )
 
-    @swagger_auto_schema(
-        request_body=LockEnkelvoudigInformatieObjectSerializer,
+    @extend_schema(
+        request=LockEnkelvoudigInformatieObjectSerializer,
         responses={
             status.HTTP_200_OK: LockEnkelvoudigInformatieObjectSerializer,
-            status.HTTP_400_BAD_REQUEST: openapi.Response(
-                "Bad request", schema=FoutSerializer
-            ),
-            status.HTTP_401_UNAUTHORIZED: openapi.Response(
-                "Unauthorized", schema=FoutSerializer
-            ),
-            status.HTTP_403_FORBIDDEN: openapi.Response(
-                "Forbidden", schema=FoutSerializer
-            ),
-            status.HTTP_404_NOT_FOUND: openapi.Response(
-                "Not found", schema=FoutSerializer
-            ),
-            status.HTTP_406_NOT_ACCEPTABLE: openapi.Response(
-                "Not acceptable", schema=FoutSerializer
-            ),
-            status.HTTP_410_GONE: openapi.Response("Gone", schema=FoutSerializer),
-            status.HTTP_415_UNSUPPORTED_MEDIA_TYPE: openapi.Response(
-                "Unsupported media type", schema=FoutSerializer
-            ),
-            status.HTTP_429_TOO_MANY_REQUESTS: openapi.Response(
-                "Throttled", schema=FoutSerializer
-            ),
-            status.HTTP_500_INTERNAL_SERVER_ERROR: openapi.Response(
-                "Internal server error", schema=FoutSerializer
-            ),
+            status.HTTP_400_BAD_REQUEST: FoutSerializer,
+            status.HTTP_401_UNAUTHORIZED: FoutSerializer,
+            status.HTTP_403_FORBIDDEN: FoutSerializer,
+            status.HTTP_404_NOT_FOUND: FoutSerializer,
+            status.HTTP_406_NOT_ACCEPTABLE: FoutSerializer,
+            status.HTTP_410_GONE: FoutSerializer,
+            status.HTTP_415_UNSUPPORTED_MEDIA_TYPE: FoutSerializer,
+            status.HTTP_429_TOO_MANY_REQUESTS: FoutSerializer,
+            status.HTTP_500_INTERNAL_SERVER_ERROR: FoutSerializer,
```

Comment:
Is deze lijst nodig? Worden deze niet uitgelezen (net zoals bij de andere ZGW componenten)?

Reply:
bij de andere componenten worden deze opgevangen door een mixin, althans bij de catalogi. Als ik dit uitzet worden ze niet gegenereerd.